testing/python/utils/test_compress_utils.py

```
@@ -1,39 +1,136 @@
+def generate_dense_input(N, trans_A, trans_B, in_dtype, seed=0):
+    torch.manual_seed(seed)
+    is_8bit = "8" in str(in_dtype)
+    is_unsigned = "uint" in str(in_dtype)
+    is_int = "int" in str(in_dtype)
+    if is_int:
+        if is_8bit:
+            low, high = (0, 128) if is_unsigned else (-64, 64)
+        else:
+            low, high = (0, 258) if is_unsigned else (-128, 128)
+        A = randint_semi_sparse(N, N, low=low, high=high, dtype=in_dtype, device="cuda", transposed=trans_A)
+        B = torch.randint(size=(N, N) if trans_B else (N, N), low=low, high=high, dtype=in_dtype, device="cuda")
```

Ruff (0.15.12)

[warning] Useless `if`-`else` condition

(RUF034)